content/en/blog/2024/sig-cloud-provider-spotlight.md

```diff
@@ -0,0 +1,143 @@
+deals directly with specific cloud provider interactions. My best advice to people wanting to do
+more work on cloud providers is to grow your familiarity with one or two cloud APIs and then look
+for open issues on the controller managers for those clouds, and always communicate with the other
+contributors as much as possible.
+
+## Accomplishments
+
+**Arujjwal:** Can you share about an accomplishment(s) of the SIG that you are proud of?
+
+**Michael:** Since I joined the SIG, more than a year ago, we have made great progress in advancing
+the extraction and migration subproject. We have moved from an alpha status on the defining
+[KEP](https://github.com/kubernetes/enhancements/blob/master/keps/README.md) to a beta status and
+are inching ever closer to removing the old provider code from the Kubernetes source tree. I've been
+really proud to see the active engagement from our community members and to see the progress we have
+made towards extraction, I have a feeling that within the next few releases, we will see the final
+removal of the in-tree cloud controllers and the completion of the subproject.
+
+## Advice for new contributors
+
+**Arujjwal:** Is there any suggestion or advice for new contributors on how they can start at SIG
+Cloud Provider?
+
+**Michael:** This is a tricky question in my opinion. SIG cloud provider is focused on the code
+pieces that integrate between Kubernetes and an underlying infrastructure. It is very common, but
+not necessary, for members of the SIG to be representing a cloud provider in an official capacity. I
+recommend that anyone interested in this part of Kubernetes should come to an SIG meeting to see how
+we operate and also to study the cloud provider framework project. We have some interesting ideas
+for future work, such as a common testing framework, that will cut across all cloud providers and
+will be a great opportunity for anyone looking to expand their Kubernetes involvement.
+
+**Arujjwal:** Are there any specific skills you're looking for that we should highlight? To give you
+an example from our own SIG Contribex: if you're an expert in [Hugo](https://gohugo.io/), we can
+always use some help with k8s.dev!
+
+**Michael:** The SIG is currently working through the final phases of our extraction and migration
+process, but we are looking toward the future and starting to plan what will come next. One of the
+big topics that the SIG has discussed is testing. Currently, we do not have a generic common set of
+tests that can be exercised by each cloud provider to confirm the behaviour of their controller
+manager. If you are an expert in Ginkgo and the Kubetest framework, we could probably use your help
+in designing and implementing the new tests.
+
+This is where the conversation ends. I hope this gave you some insights about SIG Cloud Provider's
+aim and working. This is just the tip of the iceberg. To know more and get involved with SIG Cloud
```

Comment:
```suggestion
goals and process. This is just the tip of the iceberg. To know more and get involved with SIG Cloud
```

Reply:
Why this change?

Reply:
Parallel structure, mainly. Happy to drop it if you'd rather not use it.

Reply:
@fsmunoz whichever you prefer, then.

Reply:
I'll check with the author.

Reply:
I've checked with Arrujuval and he would prefer to keep things as-is.

Reply:
Keeping as-is due to author preference.